MACSio.spec

```
@@ -93,13 +93,13 @@ do
   mkdir $mpi
   pushd $mpi
   %module_load $mpi
-  cmake -DCMAKE_INSTALL_PREFIX=%{_bindir}/$mpi \
-    -DWITH_JSON-CWX_PREFIX=%{_usr} \
+  cmake -DCMAKE_INSTALL_PREFIX=%{_libdir}/$mpi \
```

Hrm.  Interesting.  This is not actually an install _prefix_ but rather is the actual directory to install the single produced binary into.  So in that case, I think you actually want:

```suggestion
  cmake -DCMAKE_INSTALL_PREFIX=%{_libdir}/$mpi/bin \
```

Sorry.  I didn't realise how basic this package was and just assumed that was a _prefix_.

---

Done